MoMach_ros/triple_tf[] data
